src/components/Header.js

```diff
@@ -0,0 +1,31 @@
+import './Header.css';
+import PersonIcon from './PersonIcon';
+
+/* <link
+  rel="stylesheet"
+  href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0"
+/>; */
+
+const Header = () => (
+  <div className="navbar">
+    <h1 className="name">Bookstore CMS</h1>
+    <ul className="nav-links">
+      <li>
+        <a href="###" className="nav-items">
+          Books
+        </a>
+      </li>
+      <li>
+        <a href="###" className="nav-items">
+          Categories
+        </a>
+      </li>
```

Comment:
- [ ] Kindly configure these nav links to use `<Link>`, instead of the `<a>`. These links should also be refactored to work with the `react-router-dom`.